### DISCUSSION TITLE
[gatsby-starter-minimal-blog]: Cannot find reach-router module

### DISCUSSION THREAD

**Discussion by dougmellon** [LekoArts/gatsby-themes#613] [Questions] [answered]
## Description

Can not run gatsby develop upon initial instillation due to an error in which it can't find the module '@gatsbyjs/reach-router/lib/utils'.

### Steps to reproduce

> gatsby new minimal-blog LekoArts/gatsby-starter-minimal-blog

> cd minimal-blog

> gatsby develop 

### Expected result

Site should launch locally.

### Actual result

Error thrown:

> /Users/USERNAME/Desktop/minimal-blog/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
>         throw ex;
>
> Error: Cannot find module '@gatsbyjs/reach-router/lib/utils'

### Environment

System:
    OS: macOS 11.2.3
    CPU: (8) arm64 Apple M1
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.1.0 - /usr/local/bin/node
    npm: 7.11.2 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 91.0.4472.106
    Firefox: 89.0
    Safari: 14.0.3
  npmPackages:
    gatsby: ^3.6.1 => 3.7.2
    gatsby-omni-font-loader: ^1.3.1 => 1.3.1
    gatsby-plugin-feed: ^3.6.0 => 3.7.1
    gatsby-plugin-gatsby-cloud: ^2.6.1 => 2.7.1
    gatsby-plugin-google-analytics: ^3.6.0 => 3.7.1
    gatsby-plugin-manifest: ^3.6.0 => 3.7.1
    gatsby-plugin-netlify: ^3.6.1 => 3.7.1
    gatsby-plugin-offline: ^4.6.0 => 4.7.1
    gatsby-plugin-sitemap: ^4.2.0 => 4.3.1
    gatsby-plugin-webpack-bundle-analyser-v2: ^1.1.22 => 1.1.22
  npmGlobalPackages:
    gatsby-cli: 3.7.1

**Comment by LekoArts** [comment, marked as answer]
Your `npm install` didn't work correctly. Rerun `npm install --legacy-peer-deps` to install the missing packages.

**Reply by dougmellon** [reply]
Thank you for the response. Unfortunately, that did not fix the issue - same error.

**Reply by LekoArts** [reply]
Try deleting `node_modules` and `package-lock.json` then and rerun `npm install --legacy-peer-deps`

**Reply by dougmellon** [reply]
That worked perfect, thank you!
